diff --git a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
index a4f7526..cf8340b 100644
--- a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
+++ b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
@@ -468,6 +468,7 @@ class NsisTarget extends core_1.Target {
     }
     async executeMakensis(defines, commands, script) {
         const args = this.options.warningsAsErrors === false ? [] : ["-WX"];
+        args.push("-INPUTCHARSET","UTF8")
         for (const name of Object.keys(defines)) {
             const value = defines[name];
             if (value == null) {
